nearai/cli.py

```diff
@@ -474,6 +475,15 @@ def read_solutions(self, entry: str, status: Optional[bool] = None, verbose: boo
 
 
 class AgentCli:
+    def dev(self) -> None:
+        if not os.path.exists("hub/demo/.env"):
+            shutil.copy("hub/demo/.env.example", "hub/demo/.env")
+
+        ret_val = os.system("npm run dev --prefix hub/demo")
```

Comment:
Can this simply be `npm run dev`? I'm not sure what `--prefix hub/demo` is adding here unless I'm totally missing something.

We also need to run `npm i` to install dependencies. Maybe we change to `ret_val = os.system("npm i && npm run dev")`.

Also, the UI requires a minimum of Node 18 - might be worth mentioning somewhere.

I'm not sure how the Python registry works for `pip install nearai`. We'll want to make sure all of `hub/demo`'s source code is published and `nearai dev` works as expected when installed that way.